### ISSUE TITLE
problems with select all for large structures

### ISSUE THREAD

**Issue by jbloom** [dms-viz/dms-viz.github.io#95]
@WillHannon-MCB, I'm not sure if this is just an expected limitation or some sort of bug, but I thought I'd flag it.

For very large structures, such as spike, clicking the `Select All Sites` button just makes `dms-viz` freeze up and do nothing.

See for instance [this visualization](https://dms-viz.github.io/v0/?data=https%3A%2F%2Fgist.githubusercontent.com%2Fjbloom%2F978c41b85d4c03c85606ec9f222a0d32%2Fraw%2F76bf1f791b21452df63ca73e5b51b79ad02739d6%2Fdms-viz.json)

**Comment by Bernadetadad** [comment]
I don't think it freezes up it just takes a while to load and it's not obvious that it's loading.

**Comment by WillHannon-MCB** [comment]
I agree with @Bernadetadad, even though the page says 'not responding' it's still updating the structure. That said, it seems slower than normal. I don't remember it taking *this* long to highlight all the sites on a structure of Spike. I'll figure out where the slowdown is so I can fix it.
